meta-resin-common/recipes-containers/docker-disk/files/entry.sh

```diff
@@ -1,18 +1,16 @@
 #!/bin/bash
 
-#--- This script is now setup to fail if any environment variable is missing.
-
 set -o errexit
 set -o nounset
 
 DOCKER_TIMEOUT=20 # Wait 20 seconds for docker to start
 
+# Default values
 export PARTITION_SIZE=${PARTITION_SIZE:=1000}
```

Comment:
This can just be:

${PARTITION_SIZE:=1024}

No need to export it, as it is not used in a child process.
1024 is used instead of 1000 for consistency with what already defined in docker-disk.inc
Using the parameter expansion syntax ":=" there is no need to do a PARTITION_SIZE=${PARTITION_SIZE:=1000}

But ultimately, the above line can be removed altogether as there is already a check for it to be defined  in docker-disk.inc